distributed/comm/ucx.py

```diff
@@ -53,13 +53,22 @@ def init_once():
     if ucp is not None:
         return
 
+    # remove/process dask.ucx flags for valid ucx options
+    ucx_config = _scrub_ucx_config()
+    if "TLS" in ucx_config and "cuda_copy" in ucx_config["TLS"]:
+        try:
+            import numba.cuda
+        except ImportError:
+            raise ImportError(
+                "CUDA support with UCX requires Numba for context management"
+            )
+
+        numba.cuda.current_context()
```

Comment:
I think we should put a comment here explaining why we need to create a cuda context before starting communications

Reply:
Done in https://github.com/dask/distributed/pull/4787/commits/7c199af218e3808bf6e7d144986d4e951aeb134b

Reply:
Thanks @pentschev !